go/vt/worker/split_diff.go

```diff
@@ -32,9 +32,10 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/vindexes"
 	"vitess.io/vitess/go/vt/wrangler"
 
+	"sort"
```

Comment:
nit: Can you please add it to the first group above i.e. next to "sync"?

Thanks!